application/templates/application/application-form.html

```diff
@@ -5,15 +5,18 @@
 <link rel="stylesheet" type="text/css" href="{% static 'application/css/application.css' %}">
 {{ form.media.css }}
 
-<div style="margin-left:20%; height: 90%; margin-bottom: 5%; width:50%; margin-top: 2%;">
+<div style="margin-left:25%; height: 90%; margin-bottom: 2%; width:50%; margin-top: 2%;">
     {% if error_count_app %}
         <h4 class="text-danger" style="font-weight: bold">{{ error_count_app }}</h4>
     {% elif error_apply %}
         <h4 class="text-danger" style="font-weight: bold">{{ error_apply }}</h4>
     {% else %}
-    <h2>High School Application Form</h2>
-    <form action="{% url 'dashboard:application:new_application' %}" prog-data-url="{% url 'dashboard:application:ajax_load_programs' %}" method="post" class="form-signup" style="padding-top: 10px;">
+    <h2 style="font-weight: bold;">High School Application Form</h2>
+    <h5 style="color: red;margin-top: 10px;margin-bottom: 10px;">You can submit only 10 applications and only once.</h5>
```

Comment:
Not a major change but possibly changing this to say "You can submit up to 10 applications and only one submission is allowed. " would be better

Reply:
sure